source/world_builder/features/subducting_plate_models/temperature/mass_conserving.cc

```diff
@@ -169,6 +170,13 @@ namespace WorldBuilder
           prm.declare_entry("reference model name",  Types::String("half space model"),
                             "The type of thermal model to use in the mass conserving model of slab temperature. "
                             "Options are half space model and plate model");
+
+          prm.declare_entry("apply spline",  Types::Bool(false),
+                            "Whether a spline should be applied on the mass conserving model.");
+
+          prm.declare_entry("number of points in spline", Types::Int(5),
+                            "The number of points in the spline");
```

Comment:
Change "The number of points in the spline"
to "The number of points on each side of the spline centered at the point  where the two analytic equations meet. A default value of 5, will lead to a total of 11 points in the spline."